src/parsers/i3d/main.ts

```diff
@@ -110,6 +110,23 @@ function unpackData(
     numMergedMeshes: 0,
   };
   unpackPrimitives(rootSector, uncompressedValues, compressedData, maps, filterOptions);
+  const sectors = maps.idToSectorMap;
+  const numberOfTreeIndices = Math.max(maps.treeIndexNodeIdMap.length, findLargestTreeIndexInMeshes(compressedData));
+  const loadMeshesCallback = () => { return new Promise(function(resolve, reject) {
```

Comment:
This seems complicated. @nsunderland-cognite `yarn nick`

Reply:
Doesn't this do exactly the same thing as

```
const loadMeshesCallback = async () => {
  parseMeshes(...);
};
```

???